src/main/java/ch/njol/skript/events/SimpleEvents.java

```diff
@@ -599,6 +600,11 @@
 					"\t\tcancel the event",
 					"\t\tsend \"Oops! Mending failed!\" to player")
 				.since("2.5.1");
+		Skript.registerEvent("Experience Bottle Hit", SimpleEvent.class, ExpBottleEvent.class, "[e]xp[erience] bottle hit")
+			.description("Called when an experience bottle hits and releases experience.")
+			.examples("on experience bottle hit:",
+				"add random integer between 1 and 10 to experience amount")
```

Comment:
This is going to need a tab
```suggestion
				"\tadd random integer between 1 and 10 to experience amount")
```